scripts/fortran_unit_testing/run_tests.py

```diff
@@ -182,6 +182,7 @@ def cmake_stage(name, test_spec_dir, build_optimized, use_mpiserial, mpirun_comm
             "-DCIME_CMAKE_MODULE_DIRECTORY="+os.path.abspath(os.path.join(_CIMEROOT,"src","CMake")),
             "-DCMAKE_BUILD_TYPE="+build_type,
             "-DPFUNIT_MPIRUN='"+mpirun_command+"'",
+            "-DPFUNIT_PATH="+pfunit_path
```

Comment:
Do you understand why this is needed? I'm curious because it isn't needed on any of the 4 machines I've tested on. I don't remember for sure, but I thought that CMake typically picked up PFUNIT_PATH from the environment.

That said, it seems like this doesn't do any harm, so I'm okay with its being here if it truly seems to be needed.

Reply:
@billsacks I didn't have PFUNIT_PATH set in my environment. I tried setting it and that didn't help.

Reply:
I see what's going on in my unit test builds: The `Macros.cmake` file that is written in the unit test build directory has lines like this:

```cmake
  if("$ENV{MPILIB}" STREQUAL "mpich")
    if("$ENV{compile_threaded}" STREQUAL "true")
      set(PFUNIT_PATH "/usr/local/pfunit/pfunit-mpi")
    endif()
  endif()
  if("$ENV{MPILIB}" STREQUAL "mpi-serial")
    if("$ENV{compile_threaded}" STREQUAL "false")
      set(PFUNIT_PATH "/usr/local/pfunit/pfunit3.2.8-serial")
    endif()
  endif()
```

Do you not get something like that in your Macros.cmake file?

Reply:
@billsacks no, mine looks nothing like that

Reply:
I think that I know the issue - the configure generation of Macros.cmake requires the config_compilers.xml version that cesm but not acme is using - you need to update your config_compilers.xml file.

Reply:
@jgfouca Did you ever determine if this setting of `-DPFUNIT_PATH` (and most of the other changes you made in `run_tests.py`) was still needed after you got the Macros.cmake file set up appropriately? My hunch is that this should not be needed, since PFUNIT_PATH should now come from Macros.cmake

Reply:
The V1 macros file doesn't set PFUNIT_PATH, so yes.

Reply:
Okay, thanks for clarifying